File: backend/team6/src/test/java/programmers/team6/global/entity/PositiveTest.java
package programmers.team6.global.entity;

import static org.assertj.core.api.Assertions.*;

import org.junit.jupiter.api.Test;

class PositiveTest {

	@Test
	void 양수이면_정상생성(){
		assertThatCode(()->new Positive(0)).doesNotThrowAnyException();
	}

	@Test
	void 음수이면_예외발생(){
		assertThatIllegalArgumentException().isThrownBy(()->new Positive(-1));
	}
}

File: backend/team6/src/test/java/programmers/team6/global/entity/PositiveTest.kt
package programmers.team6.global.entity

import org.assertj.core.api.Assertions
import org.junit.jupiter.api.Test

internal class PositiveTest {
    @Test
    fun 양수이면_정상생성() {
        Assertions.assertThatCode { Positive(0) }.doesNotThrowAnyException()
    }

    @Test
    fun 음수이면_예외발생() {
        Assertions.assertThatIllegalArgumentException().isThrownBy { Positive(-1) }
    }
}